laika/reports.py

```diff
@@ -1145,6 +1146,22 @@ def _drive_call(self, method, *args, **kwargs):
                 else:
                     raise
 
+    def _query_arguments(self, base_arguments):
```

Comment:
I would consider to rename this method, and also the appropriate variables, to something like `_api_request_arguments`, or simply `_request_arguments` since it's used to resolve the arguments of methods that modify the destination and don't request any data whatsoever.

Reply:
good point, fixed